.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+---
+
+Please make sure to check the following for your PR — Starport Team:
+
+- [ ] I comply with the contributing.md.
```

Comment:
```suggestion
- [ ] I comply with the [`contributing.md`](contributing.md).
```

Can we even link to the file 🤔

Reply:
yes, but this makes it a bit difficult read on edit mode which is the default when you're creting a pr.

Reply:
Ok, then nvm